UPDATING.md

```diff
@@ -23,6 +23,8 @@ assists people when migrating to a new version.
 
 ## Next
 
+* [11575](https://github.com/apache/incubator-superset/pull/11575) The Row Level Security (RLS) config flag has been moved to a feature flag. To migrate, add `ENABLE_ROW_LEVEL_SECURITY: True` to the `FEATURE_FLAGS` dict in `superset_config.py`.
```

Comment:
nit: since we are changing it to a feature flag, we can rename the key to `ROW_LEVEL_SECURITY` instead of `ENABLE_ROW_LEVEL_SECURITY`

Reply:
Good point, thanks!

Reply:
G